sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1664,6 +1665,24 @@ def boundary_conditions(self):
         """
         return self._boundary_conditions
 
+    def diameter(self):
+        """
+        Returns Diameter of the beam object
+        """
+        return 1.13*sympy.sqrt(self.area)
+
+    def polar_moment(self):
+        """
+        Returns the Polar moment of inertia of the beam object
+        """
+        return 2*self.second_moment
+
+    def section_modulus(self):
+        """
+        Returns the Section modulus of the beam object
+        """
+        return (2*self.second_moment)/self.diameter()
```

Comment:
Please point to a reference that defines this. I can't remember what it is.

Reply:
@moorepants Check out this [link](https://en.wikipedia.org/wiki/Section_modulus)